dist2src/worker/monitoring.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import os
+
+from prometheus_client import CollectorRegistry, Counter, push_to_gateway
+
+logger = logging.getLogger(__name__)
+
+
+class Pushgateway:
+    def __init__(self):
+        self.pushgateway_address = os.getenv(
+            "PUSHGATEWAY_ADDRESS", "http://pushgateway"
```

Comment:
is the default actually working? is it the default DNS record for pushgatreway within the openshift deployment?

Reply:
yes it should, we use this in packit-service as well